e2e/e2etests/test_bitcoin_withdraw.go

```diff
@@ -116,8 +216,8 @@ func withdrawBTCZRC20(r *runner.E2ERunner, to btcutil.Address, amount *big.Int)
 	return rawTx
```

Comment:
Just realizing, in this test, it seems we are missing to check if cctx status is outboundMined

Reply:
Let me add it in.

Reply:
https://github.com/zeta-chain/node/pull/1942/commits/2bcdfbc727c12612159ffbf008bb00505a996b4b